src/validators/url.rs

```diff
@@ -498,7 +498,7 @@ fn check_sub_defaults(
         if let Some(default_port) = default_port {
             lib_url
                 .set_port(Some(default_port))
-                .map_err(|_| map_parse_err(ParseError::EmptyHost))?;
+                .map_err(|()| map_parse_err(ParseError::EmptyHost))?;
```

Comment:
I'm not sure why, but the linter wanted me to change this...

Reply:
You just have a newer version of rust (nightly?) than we're using in CI

Reply:
It was the CI linter that was complaining... not sure how that happened since I didn't touch this file 🤷‍♂️

Reply:
Possibly Rust 1.73 having dropped yesterday is related.

Reply:
I just merged this separately in #1010